sources/packages/web/src/views/aest/institution/Profile.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <v-card class="mt-4">
     <div class="mx-5 py-4">
-      <body-header title="Profile" class="m-1">
+      <body-header title="Profile">
```

Comment:
do we need the `div` in line 3

Reply:
if div is not given, then the whole content is moved to the left

Reply:
but looks like you already have a hood `v-card. if it the margin and padding thing you can add or adjust it in the `v-card` class

Reply:
I tried it, it didnt work
<img width="448" alt="image" src="https://user-images.githubusercontent.com/62901416/196490464-2fbbaafb-7898-47ee-8708-eb12e0958f9f.png">
So reverted back